src/spatialref.jl

```diff
@@ -616,6 +618,31 @@ function toPROJ4(spref::AbstractSpatialRef)::String
     return unsafe_string(projptr[])
 end
 
+"""
+    toEPSG(spref::AbstractSpatialRef)
+
+If possible, export EPSG code for this coordinate system.
+"""
+function toEPSG(spref::AbstractSpatialRef)::String
+    result = GDAL.osrgetauthorityname(spref.ptr, "PROJCS")
+    projcs = "PROJCS"
```

Comment:
Its confusing defining `projcs` here - why not as an `else` path of the conditional below?

Reply:
now moved